code/game/machinery/computer/dna_console.dm

```diff
@@ -262,14 +264,14 @@
 	build_genetic_makeup_list()
 
 	// Populate variables for passing to tgui interface
-	is_scramble_ready = (scrambleready < world.time)
-	time_to_scramble = round((scrambleready - world.time)/10)
+	is_scramble_ready = (scramble_ready < world.time)
+	time_to_scramble = round((scramble_ready - world.time)/10)
 
-	is_joker_ready = (jokerready < world.time)
-	time_to_joker = round((jokerready - world.time)/10)
+	is_joker_ready = (joker_ready < world.time)
+	time_to_joker = round((joker_ready - world.time)/10)
 
-	is_injector_ready = (injectorready < world.time)
-	time_to_injector = round((injectorready - world.time)/10)
+	is_injector_ready = (injector_ready < world.time)
+	time_to_injector = round((injector_ready - world.time)/10)
```

Comment:
these could probably be done better with our cooldown macros

Reply:
Where can I find examples of these?